src/runtime/composables/useSession.ts

```diff
@@ -69,7 +75,8 @@ const getCsrfToken = () => {
 }
 
 /**
- * Trigger a sign in flow for the passed `provider`. If no provider is given the sign in page for all providers will be shown.
+ * Trigger a sign in flow for the passed `provider`. If no provider is given the sign in page for all providers will be
+ * shown.
```

Comment:
nit: was this moved to a newline on accident? :D

Reply:
> So after this PR types will correctly reference what we setup in our own module, but will the approach to overwrite them by overwriting the `next-auth` types outlined here work afterwards: https://sidebase.io/nuxt-auth/recipes/custom-session-data? 🤔

Yes, it will work by overwriting the `next-auth` types. 

A little bit of context: My guess is that nuxt auto-imports was generating the result of `Session` type instead of referencing it. So I fixed it to use the `Session` from `next-auth`.